app/queries/placements/placements_query.rb

```diff
@@ -1,17 +1,32 @@
 class Placements::PlacementsQuery < ApplicationQuery
   def call
-    scope = Placement.includes(:school, :subjects)
-
+    scope = set_initial_scope
```

Comment:
Revert this back to:

```ruby
scope = Placement.includes(:school, :subjects)
```

and remove the methods you created:

- `set_initial_scope`
- `scope_with_partner_schools`
- `partner_schools_present?`